contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -228,6 +234,21 @@ In this case, only the `root` component can be accessed in Vue while the rest ar
 
 <br>
 
+7. **Lifecycle Hooks**<br>
+    Every Vue instance goes through a series of initialization steps when it is created, i.e. setting up data observation, compiling the template, mounting the instance to the DOM, and updating the DOM when data modifies. Along these steps, Vue runs functions called lifecycle hooks, allowing users to add code at specific steps.<br/><br/>
```

Comment:
Along these steps, Vue runs functions **in the background** called lifecycle hooks, allowing users to add code ~at specific steps~ **at each stage that could improve its rendering speed.**